ui/src/composables/private/__tests__/use-model-toggle.cy.js

```diff
@@ -165,39 +165,53 @@ describe('use-model-toggle API', () => {
           .then(() => {
             expect(fn).not.to.be.called
           })
-        cy.get('body')
-          .click(499, 0)
+
         cy.dataCy('menu')
-          .should('not.exist')
-          .then(() => {
-            expect(fn).to.be.called
+          .should('exist').then(() => {
```

Comment:
Remove `.should('exist')` where you added it, as it's implicit in `.dataCy` which is based upon `.get`